service/hook/github/github.go

```diff
@@ -325,8 +330,13 @@ func (hp HookProvider) TransformRequest(r *http.Request) hookCommon.TransformRes
 
 // returns the repository clone URL depending on the publicity of the project
 func (branchInfoModel BranchInfoModel) getRepositoryURL() string {
-	if branchInfoModel.Repo.Private {
-		return branchInfoModel.Repo.SSHURL
+	return branchInfoModel.Repo.getRepositoryURL()
+}
+
+// returns the repository clone URL depending on the publicity of the project
```

Comment:
This comment was a bit obscure in the first place, I'd rather not duplicate it. If something, I'd extend it with mentioning the SSH URL too, but I'm fine removing it completely, as the implementation is quite trivial and readable.